bitnami/gen-yaml/gen

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+mkdir -p ${NS}/upstream
+
+# Fetch from helm repo: https://github.com/bitnami/charts/tree/main/bitnami/kafka#installing-the-chart
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm template ${BITNAMI_KAFKA_RELEASE} bitnami/kafka --version ${BITNAMI_KAFKA_RELEASE} -f ${VALUES} --set fullnameOverride=${NAME} --set nameOverride=${NAME} --set kraft.clusterId=${KRAFT_CLUSTERID} --set auth.tls.existingSecrets="{${NAME}-0-tls,${NAME}-1-tls,${NAME}-2-tls}" --namespace ${NS} > ${NS}/upstream/kafka.yaml
+
+cat gen-yaml/clean-upstream-overlay.yaml > ${NS}/upstream/kustomization.yaml
+
+if [ ! -f ${NS}/kustomization.yaml ]; then
+cat gen-yaml/configure-logging-overlay.yaml > ${NS}/kustomization.yaml
```

Comment:
I think we're using configure-logging-overlay.yaml more like the initial "final" kustomization file. I think we should rename it to something like "namespace-kustomization-template.yaml" or "instance-kustomization-stub.yaml".

Reply:
Yes, not the best name - how about `namespace-overlay.yaml` to keep it consistent with `clean-upstream-overlay.yaml`?

Reply:
renamed